includes/data-stores/class-wcs-subscription-data-store-cpt.php

```diff
@@ -4,7 +4,7 @@
 }
 
 /**
- * Subscription Data Store: Stored in CPT.
+ * Subscription Data Store: Stored in CPT (posts table).
```

Comment:
I added this to clarify: `CPT` was too similar to `COT` and it confused me for a minute 😄